rust/routee-compass-core/src/model/road_network/turn.rs

```diff
@@ -0,0 +1,32 @@
+use super::graph_error::GraphError;
+
+pub enum Turn {
+    NoTurn,
+    SlightRight,
+    SlightLeft,
+    Right,
+    Left,
+    SharpRight,
+    SharpLeft,
+    UTurn,
+}
```

Comment:
seems we could make Turn deserializable and then expose an optional `HashMap<Turn, Time>` mapping to some turn delays configuration in the future to remove the hard-coded values in the energy traversal model

Reply:
Yeah it would be nice to expose the turn cost to the configuration. I struggled to land on a way to do this that felt right and so maybe we could have a design discussion about ways to abstract this.